cmd/lotus-shed/cid.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/ipfs/go-cid"
+	mh "github.com/multiformats/go-multihash"
+)
+
+var cidCmd = &cli.Command{
+	Name: "cid",
+	Subcommands: cli.Commands{
+		cidIdCmd,
+	},
+}
+
+var cidIdCmd = &cli.Command{
+	Name:  "id",
+	Usage: "create identitg CID from hex or base64 data",
+	Action: func(cctx *cli.Context) error {
+		if !cctx.Args().Present() {
+			return fmt.Errorf("must specify data")
+		}
+
+		dec, err := hex.DecodeString(cctx.Args().First())
```

Comment:
technically you can have a string be valid in both hex and base64, can't you?

Reply:
s/identitg/identity/

Reply:
Highly unlikely unless you specifically craft one